backend/main.go

```diff
@@ -11,6 +11,8 @@ func main() {
 	database.ConnectDatabase()
 	database.SeedUsers()
```

Comment:
Probably at this point might be worth throwing these all into a `database.Seed()` function and wrap these 4 in their.

Purely an organisational move before main.go gets polluted with seed calls.

Instead of reading it as:

- Connect to Database
- Seed Users
- Seed Tierlists
- Seed Tiers
- Seed Items
- Setup Gin / Routes
- Run HTTP Server

We can simplify to:

- Connect to Database
- Seed Data
- Setup Gin / Routes
- Run HTTP Server

Can then stay this way and we can keep adding more seeding functions to `database.Seed()`